import_models_to_verta/mlflow_databricks_bulk_import_example.ipynb

```diff
@@ -0,0 +1,341 @@
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "mlflow_models: List[MLflow_RegisteredModel]= list()\n",
+    "\n",
+    "result: PagedList = mlflow_client.search_registered_models()\n",
+    "mlflow_models += result.to_list()\n",
+    "while result.token:  # handle paginated results\n",
+    "    result: PagedList = mlflow_client.search_registered_models(\n",
+    "    page_token=result.token\n",
+    "    )\n",
+    "    mlflow_models += result.to_list()\n",
+    "\n",
+    "mlflow_models.sort(key=lambda x: x.name)\n",
+    "for mv in mlflow_models:\n",
+    "    print(f\"name={mv.name}; version={mv.version}\")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Reduce the list if desired"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Example: filter out models with \"test\" in the name\n",
+    "models_for_import = [model for model in mlflow_models if \"test\" not in model.name]"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Import Models to Verta"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# These are the names of files that Verta uses internally and cannot be overwritten\n",
+    "protected_file_names = [\n",
+    "    \"custom_modules\",\n",
+    "    \"model\",\n",
+    "    \"model.pkl\",\n",
+    "    \"model_api.json\",\n",
+    "    \"requirements.txt\",\n",
+    "    \"train_data\",\n",
+    "    \"tf_saved_model\",\n",
+    "    \"setup_script\",\n",
+    "]"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "failed_model_imports: List[MLflow_RegisteredModel] = list()\n",
+    "successful_model_imports: List[MLflow_RegisteredModel] = list()\n",
+    "model_messages: Dict[MLflow_RegisteredModel, List[str]] = dict()\n",
+    "\n",
+    "for rm in models_for_import:\n",
+    "    model_messages[rm] = list()\n",
+    "\n",
+    "    try:\n",
+    "        verta_rm = verta_client.create_registered_model(\n",
+    "            name=rm.name,\n",
+    "            desc=rm.description,\n",
+    "            # Note: hacking into labels because we don't have a way to add attributes to registered models\n",
```

Comment:
I'd recommend not calling something out as a hack in an end-user facing notebook; it hurts our credibility. We should be fine to omit this comment.